install.py

```diff
@@ -9,7 +9,10 @@
 
 def build(library):
     print('Building %s...' % library)
-    client.images.build(path=os.getcwd(), tag='ann-benchmarks-%s' % library, rm=True, dockerfile='install/Dockerfile.%s' % library)
+    try:
+        client.images.build(path=os.getcwd(), tag='ann-benchmarks-%s' % library, rm=True, dockerfile='install/Dockerfile.%s' % library)
+    except docker.errors.BuildError as err:
+        print("Build error: {0}".format(err))
```

Comment:
what's the rationale for continuing on errors? don't have strong feelings, but seems unrelated to the rest of the changes

Reply:
Ah, I've added it after (for some reason) the faiss container couldn't be build, and everything following faiss couldn't be installed as a result. (They've released a new version, maybe I pulled somewhere in between where it didn't compile.) 

Since we don't have control over the repos that we are using, it's likely that once in a while a container cannot be build. It shouldn't stop all the other containers from getting installed, should it?

Reply:
that's a valid point, makes sense